vnext/Test/React.Windows.Test.vcxproj

```diff
@@ -1,5 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="16.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
```

Comment:
This seems unusual.
Does the `Globals` section really need to go before `ProjectConfigurations`?

Reply:
Without it, VS doesn't load the property pages properly. They're full of blanks. Everything works, but it's confusing.